test/test-v8debugapi.ts

```diff
@@ -870,9 +906,11 @@ describe('v8debugapi', function() {
           assert((bVal as any).members[1].name.match(
             'Only first.*config.capture.maxProperties=1'));
 
-          api.clear(bp);
           config.capture.maxProperties = oldMax;
```

Comment:
The same concern about moving this line inside the callback of `api.clear` as described above.